master/internal/stream/trials.go

```diff
@@ -139,7 +139,8 @@ func TrialCollectStartupMsgs(
 		Table("trials").
 		Column("trials.id").
 		Join("JOIN experiments e ON trials.experiment_id = e.id").
-		Join("JOIN projects p ON e.project_id = p.id")
+		Join("JOIN projects p ON e.project_id = p.id").
+		OrderExpr("id ASC")
```

Comment:
nit: add the table this field is coming from so it's clear which `id` is being used
```suggestion
		OrderExpr("trials.id ASC")
```